src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,7 +25,10 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.Uap, 
```

Comment:
This is too confusing now: we need to figure out better names for UWP.
@danmosemsft Why are we not calling these UWP which is the [official name ](https://docs.microsoft.com/en-us/windows/uwp/layout/design-and-ui-intro) now?